test/can-view-parser-test.js

```diff
@@ -784,8 +784,8 @@ test('camelCase properties are encoded with on:, :to, :from, :bind bindings', fu
 testHelpers.dev.devOnlyTest('Warn on missing attribute value end quotes (canjs/can-view-parser#7)', function () {
 	var makeWarnChecks = function(input, texts) {
 		var count = 0;
-		var teardown = testHelpers.dev.willWarn(/End quote is missing for/, function(matched, text) {
-			equal(matched, texts[count++]);
+		var teardown = testHelpers.dev.willWarn(/End quote is missing for/, function(message, matched) {
+			ok(matched, texts[count++]);
```

Comment:
You can just not pass a function here since the `equal(count, teardown())` should handle checking that it was called the correct number of times.

Reply:
So ```count``` will be hard-coded and assigned to 1, otherwise the tests will fail.

Reply:
Yeah, you could just do `equal(1, teardown())`. That would make these easier to read I think.